processor/batchprocessor/batch_processor.go

```diff
@@ -234,6 +235,7 @@ func (bp *batchMetricProcessor) sendItems(measure *stats.Int64Measure) {
 	statsTags := []tag.Mutator{tag.Insert(processor.TagProcessorNameKey, bp.name)}
 	_ = stats.RecordWithTags(context.Background(), statsTags, measure.M(1))
 	_ = stats.RecordWithTags(context.Background(), statsTags, statBatchSendSize.M(int64(bp.batchMetrics.metricData.MetricCount())))
+	_ = stats.RecordWithTags(context.Background(), statsTags, statBatchSendSizeBytes.M(int64(bp.batchMetrics.metricData.Size())))
```

Comment:
Please combine all Records in one.

Reply:
done

Reply:
done https://github.com/open-telemetry/opentelemetry-collector/issues/1413